src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,109 @@
+import { randomBytes } from 'crypto';
+import user from '../db/models';
```

Comment:
This should be ...

```js
import models from '../db/models';
``` 
 ... because on this line, you're importing all the models in that directory. Naming all the models on the app `users` doesn't sound descriptive enough now, does it? 🙂 

You can then go ahead to destructure the `User` model on line 5 like so:

```js
const { User } = model;
```

Reply:
resolved